ros/angel_msgs/msg/SystemCommands.msg

```diff
@@ -9,6 +9,6 @@
 #
 
 # Task Monitor commands
-bool reset_current_task false
-bool previous_step      false
-bool next_step          false
+bool reset_current_task
+bool previous_step
+bool next_step
```

Comment:
Did the default values not work, or cause something else to not work?

Reply:
Yeah, they were causing compiler error when I tried to generate the C# scripts. It seemed like that the ROS Unity library didn't support the default syntax.